insolar/mainnet
# MN-651: split spec files

**- What I did**
I split the api-exported.yaml file into two files: api-exported.yaml ( with external API ) and api-exported-internal.yaml ( with internal API )
**- How I did it**

- Added new file api-exported-internal.yaml from spec-insolar-platform repo (/internal-call-api/client/api-exported.yaml) tag [v1.1.1](https://github.com/insolar/spec-insolar-platform/tree/v1.1.1)
- Replaced content of old api-exported.yaml to content of /external-call-api/client/api-exported.yaml tag [v1.1.1](https://github.com/insolar/spec-insolar-platform/tree/v1.1.1)
- Updated submodule insolar-scripts to [51c05edbe14104f9582998da60b3b9a972177c95](https://github.com/insolar/insolar-scripts/commit/51c05edbe14104f9582998da60b3b9a972177c95)

**- How to verify it**

- Verify `paths` into the api-exported.yaml and api-exported-internal.yaml. They should contain only the corresponding values for each.
- Run fund tests

**- Description for the changelog**
